### DISCUSSION TITLE
What is peer and seed in hermes relayer?

### DISCUSSION THREAD

**Discussion by Venusdev2369** [informalsystems/hermes#2684] [Q&A] [answered]
## Question
Hello, I've been setting up the hermes relayer between my chain and cosmos hub.
But I don't know the exact meaning of peer and seed.
Please give me some explanations about peer and seed.

**Comment by romac** [comment]
We do not use this terminology in Hermes but rather talk about:
- source and destination chain when talking about the relaying flow of a packet
- host and reference chain when talking about IBC clients

May I ask where you have seen mention of _peer_ and _seed_ in relation with Hermes?

In any case, for any pair of chains specified in the configuration, Hermes will relay packets in _both directions_ between the chains.

Hope that helps!

**Comment by Venusdev2369** [comment]
Thank you for your reply.
If I am going to run my node, I must give the seeds information to app.toml file.
Here I mean seed node and peer node

**Comment by adizere** [comment]
For starting & operating a full node, you will need to look at network specific documentation. For example, if you want to operate an Osmosis node, start here: https://docs.osmosis.zone. For Evmos, check this out: https://docs.evmos.org/validators/mainnet.html. Each network has their specific details, which include seed node details.

There is also the [chain registry](https://github.com/cosmos/chain-registry), which include publicly-accessible seed nodes for all networks, examples for [akash](https://github.com/cosmos/chain-registry/blob/61ea7bed445fc8b7163dde02b1d318f306469efb/akash/chain.json#L41)

**Comment by Venusdev2369** [comment]
Thank you for your reply.
It helps me a lot but I have some questions.
If I set up the relayer between my chain and cosmos hub, do I have to set up the relayer on the machine running my node?
and why do I have set the daemon settings?

**Comment by adizere** [comment, marked as answer]
> do I have to set up the relayer on the machine running my node?

Not necessarily, the relayer and your network can be on separate machines.

> and why do I have set the daemon settings?

I don't understand, would you mind rephrasing? See also https://docs.osmosis.zone/osmosis-core/relaying/relayer-guide/
